Makefile

```diff
@@ -2,10 +2,14 @@
 # Macros #
 ##########
 
+# --- Release Version --- #
+RELEASE_VERSION = 0.3.0
```

Comment:
Add section in the Wiki page with instructions for building Spark components - with pre-requisites etc, running test program etc